table/table.cpp

```diff
@@ -75,37 +73,16 @@ ::database::Row Table::get_row(string &column, ::database::Entry comparison) {
     try {
         index = column_index(column);
     } catch (std::exception e) {
-        // catch code
-        // throw()
+        throw e;
```

Comment:
What's the point in re-throwing `e` after catching it? If an exception is thrown from `column_index`, it would get thrown anyways to the caller of this method, right?

Consider either removing the try-catch block around `index = column_index(column);` if you do intend to just re-throw the same exception.

If you want to add extra behavior before re-throwing (e.g. logging an error message from here for developers to see and not the user), then please do that here in this PR. 

Also, if there is a specific exception you expect to be thrown from `column_index`, then consider catching that specific exception in the catch block. instead of the general `std::exception`.